src/VIMLoader.ts

```diff
@@ -355,6 +355,7 @@ export class VIMLoader {
       )
 
       resultMesh.computeBoundingSphere()
+      resultMesh.computeBoundingBox()
```

Comment:
Is necessary to call both computeBoundingBox and computeBoundingSphere? The boundingSphere is trivially derived from the bounding box, but I am concerned that the implementation of each iterate over the vertices.